Lib/test/test_builtin.py

```diff
@@ -1626,8 +1626,12 @@ def test_sum(self):
 
         self.assertEqual(sum(range(10), 1000), 1045)
         self.assertEqual(sum(range(10), start=1000), 1045)
+        self.assertEqual(sum(range(10), 0.1), 45.1)
```

Comment:
I don't think that adding new test cases about `sum` is part of this PR.
If you want to improve `sum` tests, you can open separate issue/PR.

However, as a rule, float objects is not compared  by `assertEqual`. You should use `assertAlmostEqual`

Reply:
Fixed. I will move them to another PR if that would be better.